src/robot/libraries/Telnet.py

```diff
@@ -14,6 +14,7 @@
 
 from contextlib import contextmanager
 import inspect
+import socket
```

Comment:
Nitpick: Imports are ordered alphabetically. This import should be before `import struct`.